src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpMetadataContext.cs

```diff
@@ -9,27 +9,38 @@ namespace Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator
     internal struct CSharpMetadataContext
     {
         internal readonly ImmutableArray<MetadataBlock> MetadataBlocks;
+        internal readonly ImmutableArray<Alias> Aliases;
         internal readonly CSharpCompilation Compilation;
         internal readonly EvaluationContext EvaluationContext;
 
-        internal CSharpMetadataContext(ImmutableArray<MetadataBlock> metadataBlocks, CSharpCompilation compilation)
+        internal CSharpMetadataContext(
+            ImmutableArray<MetadataBlock> metadataBlocks, 
+            ImmutableArray<Alias> aliases,
```

Comment:
Should the context depend on aliases? Won't this mean the context will not be re-used across expressions and call stack, or when stepping when the set of { $ReturnValue } changes?